packages/identity-wallet-mobile/src/setup-redux.js

```diff
@@ -0,0 +1,5 @@
+import '@selfkey/wallet-core/redux/middlewares/register-redux-logger';
+import '@selfkey/wallet-core/redux/middlewares/register-redux-thunk';
+// We can enable redux saga when needed
```

Comment:
I would prefer not to use redux - saga
redux thunk and and redux promise cover everything we need. 

saga is hard to manage once complexity raises

Reply:
Sure, i agree. I will remove redux-saga from dependencies and add the redux promise middleware that is missing.